api/applications/views.py

```diff
@@ -98,3 +105,31 @@ def perform_update(self, serializer):
         serializer.validated_data['owner'] = self.request.user
         # TODO: Write code to transfer ownership
         serializer.save(owner=self.request.user)
+
+
+class ApplicationReset(generics.RetrieveUpdateDestroyAPIView, ApplicationMixin):
```

Comment:
Ok, that makes sense. If this view is only for updating the client secret, should it be `generics.UpdateAPIView` instead?